### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by Manethpak**
Issue Number: #4002

## PR Details

-  Change Version to 1.0.1
-  Change in Version section `The feature set is listed above in the Features section` with attach link
-  Change the current link for Create bug report and Submit feature request to their corresponding issue template provided.

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ The server exposes a GraphQL API for all actions. The server is built with the f
 Amplication Client is the front end of the platform that provides you with an easy-to-drive UI for building your next low-code application.
 The client is based on React, Apollo client, Primer components, React Material Web Components, Formik, and more.
 
-# Version 1.0.0
+# Version 1.0.1
```
Comment (souravjain540) [inline README.md:123]: Let's do this as Version 1 only so we don't have to change it on every minor and patch release.

File: README.md [hunk 2]
```diff
@@ -120,9 +120,9 @@ The server exposes a GraphQL API for all actions. The server is built with the f
...
 
-Amplication is currently in its version 1.0.  This is the first major release of Amplication with enterprise-grade production readiness & scale. In this version, we have introduced multiple new features and enhanced the existing ones. The feature set is listed above in the README file. 
+Amplication is currently in its version 1.0.  This is the first major release of Amplication with enterprise-grade production readiness & scale. In this version, we have introduced multiple new features and enhanced the existing ones. The feature set is listed above in the [Features](#features) section.
```
Comment (souravjain540) [inline README.md:125]: loved this :)

File: README.md [hunk 3]
```diff
@@ -132,12 +132,12 @@ Ask your questions and participate in discussions on Amplication-related and web
 
 ## Create a bug report
 
-If you see an error message or run into an issue, please [create bug report](https://github.com/amplication/amplication/issues/new?assignees=&labels=type%3A%20bug&template=bug_report.md&title=). This effort is valued and it will help all Amplication users.
+If you see an error message or run into an issue, please [create bug report](https://github.com/amplication/amplication/issues/new?assignees=&labels=type%3A+bug&template=bug.yaml&title=%F0%9F%90%9B+Bug+Report%3A+). This effort is valued and it will help all Amplication users.
```
Comment (souravjain540) [inline README.md:135]: perfect

File: README.md [hunk 4]
```diff
@@ -132,12 +132,12 @@ Ask your questions and participate in discussions on Amplication-related and web
...
 
 
 ## Submit a feature request
 
-If you have an idea, or you're missing a capability that would make development easier and more robust, please [Submit feature request](https://github.com/amplication/amplication/issues/new?assignees=&labels=type%3A%20feature%20request&template=feature_request.md&title=).
+If you have an idea, or you're missing a capability that would make development easier and more robust, please [Submit feature request](https://github.com/amplication/amplication/issues/new?assignees=&labels=type%3A+feature+request&template=feature.yml).
```
Comment (souravjain540) [inline README.md:140]: awesome!

**Review by souravjain540 — CHANGES_REQUESTED** [review]
Really good changes. Just make the changes I requested and I will approve and merge this PR :)

File: README.md (hunk 1 above)
Comment (Manethpak) [reply README.md]: Made the change!

**Review by souravjain540 — APPROVED** [review]
Well done!

**Pull request merged**